packages/cactus-plugin-ledger-connector-fabric/src/main/json/openapi.json

```diff
@@ -286,6 +286,42 @@
                         "type": "string"
                     }
                 }
+            },
+            "PrometheusExporterOptions": {
+                "type": "object",
+                "required": [
+                    "metricsPollingIntervalInMin"
+                ],
+                "properties": {
+                    "metricsPollingIntervalInMin": {
+                        "description": "The polling interval for the prometheus exporter (in minutes)",
+                        "type": "number",
+                        "nullable": false
+                    }
+                }
+            },
+            "PrometheusExporterMetricsRequest": {
+                "type": "object",
+                "required": [
+                    "promExporter"
+                ],
+                "properties": {
+                    "promExporter":{
```

Comment:
@jagpreetsinghsasan Do we still need this in the request? IIRC HTTP GET requests cannot have a body.

Reply:
Fixed